experimental/Matrix/matrix.jl

```diff
@@ -135,6 +135,26 @@ function _lex_isEqual(a::T,b::T) where T<:MatElem
   return true
 end
 
+function size(G::GapObj)
+    return fmpz(GAP.Globals.Size(G))
+end
+
+function maximalsubgroups(G::GapObj)
+    return GAP.Globals.MaximalSubgroups(G)
+end
+
+function smallgeneratingset(G::GapObj)
+    return GAP.Globals.SmallGeneratingSet(G)
+end
+
+function isomorphismpermgroup(G::GapObj)
```

Comment:
Please take a look at the oscar manual, many of these functions already exist, possibly under slightly different names